src/components/legend/defaults.js

```diff
@@ -147,4 +151,31 @@ module.exports = function legendDefaults(layoutIn, layoutOut, fullData) {
 
         Lib.coerceFont(coerce, 'title.font', dfltTitleFont);
     }
+
+    return true;
```

Comment:
?

Reply:
Good eye. This was a leftover from initial commits.
Resolving.